VibeSomeMauiBro.UITests/MainPageTests.cs

```diff
@@ -0,0 +1,99 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+
+namespace VibeSomeMauiBro.UITests;
+
+public class MainPageTests : BaseTest
+{
+    private const int WaitForContentLoadMs = 2000;
+    [Fact]
+    public void App_Should_LaunchWithoutCrashing()
+    {
+        try
+        {
+            // Arrange & Act
+            InitializeAndroidDriver();
+            
+            // Give the app a moment to fully initialize
+            Thread.Sleep(3000);
+            
+            // Assert - If we get here without an exception, the app launched successfully
+            Assert.NotNull(Driver);
+            Assert.True(Driver.SessionId != null, "Driver should have a valid session");
+            
+            // Verify the driver is responsive and the current activity matches expected
+            var currentActivity = Driver.CurrentActivity;
+            Assert.Equal(ActivityName, currentActivity);
```

Comment:
This is really close:
```
  Failed VibeSomeMauiBro.UITests.MainPageTests.App_Should_LaunchWithoutCrashing [38 s]
  Error Message:
   Assert.Equal() Failure: Strings differ
           ↓ (pos 0)
Expected: "com.companyname.vibesomemauibro.MainActiv"···
Actual:   ".MainActivity"
           ↑ (pos 0)
```

If we make `ActivityName` just `.MainActivity`, it should fix it?